pkg/containers/containers.go

```diff
@@ -271,7 +270,7 @@ const (
 	containerStarted
 )
 
-func (c *Containers) PopulateBpfMap(bpfModule *bpf.Module) error {
+func (c *Containers) PopulateBpfMap(bpfModule *libbpfgo.Module) error {
 	containersMap, err := bpfModule.GetMap("containers_map")
```

Comment:
containers_map shouldn't be hard-coded if we move this to a package.
Let's change this to get the map name as an additional arugment

Reply:
Agree